templates/about.html

```diff
@@ -10,10 +10,20 @@
             <a href="/about">About</a>
         </nav>
     </header>
-    <div>
-      <h1>Background</h1>
-      <p><a href="https://github.com/w3c/web-platform-tests">web-platform-tests</a> is a test suite for the web platform.</p>
-    </div>
+    <article>
+      <h1>About</h1>
+      <p>The WPT Dashboard, <a href="http://wpt.fyi/">wpt.fyi</a>, displays results for the <a href="https://github.com/w3c/web-platform-tests">web-platform-tests</a>, or WPT, which are a group of test suites for many web platform specifications (each top-level directory of WPT is a different specification).</p>
```

Comment:
I just removed the parenthetical since that detail is mentioned at the beginning of the WPT README and docs.